examples/rust-data-loader/rust/src/lib.rs

```diff
@@ -69,50 +66,39 @@ impl DataLoader for NDJsonLoader {
         self.rows.replace(rows);
         console::log(&format!["Temperature[{temperature_count}], Accelerometer[{accelerometer_count}]"]);
 
-        let accelerometer_channel = Channel {
-            id: ACCELEROMETER_CHANNEL_ID,
-            schema_id: Some(ACCELEROMETER_SCHEMA_ID),
-            topic_name: "/accelerometer".to_string(),
-            message_encoding: Vector3::get_message_encoding(),
-            message_count: Some(accelerometer_count as u64),
-        };
-        let accelerometer_schema = Schema::from_id_sdk(
-            ACCELEROMETER_SCHEMA_ID,
-            Vector3::get_schema().ok_or(anyhow!["failed to get Vector3 schema"])?,
-        );
+        let mut init = Initialization::builder()
+            .start_time(seconds_to_nanos(start_seconds))
+            .end_time(seconds_to_nanos(end_seconds));
 
-        let temperature_channel = Channel {
-            id: TEMPERATURE_CHANNEL_ID,
-            schema_id: Some(TEMPERATURE_SCHEMA_ID),
-            topic_name: "/temperature".to_string(),
-            message_encoding: Temperature::get_message_encoding(),
-            message_count: Some(temperature_count as u64),
-        };
-        let temperature_schema = Schema::from_id_sdk(
-            TEMPERATURE_SCHEMA_ID,
-            Temperature::get_schema().ok_or(anyhow!["failed to get Temperature schema"])?
-        );
+        let mut accelerometer_schema = init.add_schema::<Vector3>()?;
+        let accelerometer_channel = accelerometer_schema
+            .add_channel("/accelerometer")
```

Comment:
it feels a bit awkward to have `schema.add_channel(...)`. I would expect something like `init.add_channel("/accelerometer").schema(accelerometer_schema)...` instead